### ISSUE TITLE
ONNX conversion

### ISSUE THREAD

**Issue by farazBhatti** [hughplay/DFNet#4]
Hi ,is it possible to convert DFNet pytorch model to ONNX format?

**Comment by hughplay** [comment]
Hi Faraz Ahmad,

I have no experience about transferring PyTorch models into ONNX format before.
But it seems available code of model and pre-trained weights is enough for transferring according to the official document.

Official tutorial: https://pytorch.org/tutorials/advanced/super_resolution_with_caffe2.html
Model: https://github.com/hughplay/DFNet/blob/master/model.py
Pre-trained weights: https://drive.google.com/drive/folders/1lKJg__prvJTOdgmg9ZDF9II8B1C3YSkN?usp=sharing

Let me know if you need some extra helps.

**Comment by farazBhatti** [comment]
@hughplay , so if I want to convert your pytorch model into coreml would it be possible to do it . like does your model have any custom layer implemented which might hinder in pytorch to coreml conversion?

**Comment by akash1301** [comment]
Hi, I have converted the pre-trained model into onnx format. But due to a lot of bugs, onnx to core ml conversion is not possible. The latest coremltools 4.0b2 supports conversion from pytorch to core ml.  I tried until I faced that [issue](https://github.com/apple/coremltools/issues/827?fbclid=IwAR1TveXCdE2r62yP_d_RXCAjyWtNxHyR1aBg5UqhTA05-qyf-8BCtuaX6Ys) . Then tried by downgrading the pytorch version and got this error, 

> RuntimeError: PyTorch convert function for op 'constant_pad_nd' not implemented.

It seems there are some op that is not implemented. Can it be customized to convert it into coreml?
